packages/trader/src/Modules/MT5/Components/mt5-personal-details-form.jsx

```diff
@@ -81,27 +81,21 @@ const validatePersonalDetails = ({
         account_opening_reason: localize('Account opening reason'),
     };
 
-    const common_messages = [
-        localize('{{field_name}} is required'),
-        localize('{{field_name}} is not properly formatted.'),
-    ];
+    const common_messages = ['{{field_name}} is required', '{{field_name}} is not properly formatted.'];
 
     const errors = {};
 
     Object.entries(validations).forEach(([key, rules]) => {
         const error_index = rules.findIndex(v => !v(values[key]));
         if (error_index !== -1) {
-            switch (key) {
-                default:
-                    errors[key] = errors[key] = (
-                        <Localize
-                            i18n_default_text={common_messages[error_index]}
-                            values={{
-                                field_name: mappedKey[key],
-                            }}
-                        />
-                    );
-            }
+            errors[key] = errors[key] = (
+                <Localize
+                    i18n_default_text={common_messages[error_index]}
```

Comment:
as far as I know our translation doesn't work with variables and the localize needs to be around the string itself